css/style.css

```diff
@@ -1,5 +1,5 @@
 *{
-
+  color: white;
 }
```

Comment:
I made some cosmetic changes while debugging, so it looks like the changes were a lot bigger than they were. I've marked the most useful ones with line comments like this, further down.